pkg/coscheduling/core/core.go

```diff
@@ -94,13 +94,16 @@ func NewPodGroupManager(pgClient pgclientset.Interface, snapshotSharedLister fra
 		pgLister:             pgInformer.Lister(),
 		podLister:            podInformer.Lister(),
 		permittedPG:          gochache.New(3*time.Second, 3*time.Second),
-		podGroupBackoff:      gochache.New(10*time.Second, 10*time.Second),
+		backedOffPG:          gochache.New(10*time.Second, 10*time.Second),
 	}
 	return pgMgr
 }
 
-func (pgMgr *PodGroupManager) AddToPodGroupBackoff(pgName string) {
-	pgMgr.podGroupBackoff.Add(pgName, nil, 10*time.Second)
+func (pgMgr *PodGroupManager) BackoffPodGroup(pgName string, backoff time.Duration) {
+	if backoff == time.Duration(0) {
```

Comment:
commented elsewhere: IMO we should return as early as possible.

Reply:
we can check whether `cs.pgBackoff != nil` outside, It's already done